.github/workflows/version-bump.yml

```
@@ -17,6 +17,14 @@ jobs:
       - name: Checkout Branch
         uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
 
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v5
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
```

Instead of copying and updating these values across each repository, how do we feel about retrieving the secrets from our production Key Vault like our other workflows?

---

That would require additional PRs to each repository and also deleting the secrets from each repository.  To me, that feels like a lot of work when the secrets can be added to the production Key Vault once and then use the following snippet:

```
- name: Login to Azure - Prod Subscription
  uses: Azure/login@1f63701bf3e6892515f1b7ce2d2bf1708b46beaf
  with:
    creds: ${{ secrets.AZURE_PROD_KV_CREDENTIALS }}

- name: Retrieve secrets
  id: retrieve-secrets
  uses: bitwarden/gh-actions/get-keyvault-secrets@c3b3285993151c5af47cefcb3b9134c28ab479af
  with:
    keyvault: "bitwarden-prod-kv"
    secrets: "github-gpg-private-key,
      github-gpg-private-key-passphrase"
```

You can then change the existing lines to look like this:

```suggestion
          gpg_private_key: ${{ steps.retrieve-secrets.outputs.github-gpg-private-key }}
          passphrase: ${{ steps.retrieve-secrets.outputs.github-gpg-private-key-passphrase }}
```